osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs

```diff
@@ -42,6 +42,9 @@ public partial class BarHitErrorMeter : HitErrorMeter
         [SettingSource("Label style", "How to show early/late extremities")]
         public Bindable<LabelStyles> LabelStyle { get; } = new Bindable<LabelStyles>(LabelStyles.Icons);
 
+        [SettingSource("Bar visibility")]
+        public Bindable<bool> BarVisibility { get; } = new Bindable<bool>(true);
```

Comment:
Is it just me or is this really non-descript? I'd probably go for "Show colour bars" or similar for the name of this setting.

Reply:
I didn't feel it to be as you say at first, but better safe than sorry so addressed.